src/autoschedulers/anderson2021/generate_data.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
+
+# This script will generate a batch of data using the autoscheduler, retraining
+# the cost model after each batch. It can be used for generating training data or
+# for autotuning on an individual app.
+# It is a wrapper around autotune_loop.sh, which handles compiling, benchmarking,
+# and retraining the cost model. This file makes the process more user friendly
+# by providing statistics, support for resuming previous batches, autotuning
+# across multiple apps, etc.
+#
+# It assumes that the autoscheduler itself and any apps to be autoscheduled have
+# already been built and the resulting files are stored in halide_build_dir.
+# Using CMake is recommended because this script assumes that the given
+# halide_build_dir has the same structure that the CMake build will produce
+#
+# Arguments:
+# halide_build_dir - path where Halide is built
+# max_iterations - the number of batches to generate. The cost model is
```

Comment:
Maybe add a set of valid inputs for each argument to this shell script, e.g. `train_only [0|1] - don't generate...` or something along the line as it makes using this script a little easier